mps_youtube/g.py

```diff
@@ -51,6 +51,16 @@
 HISTFILE = os.path.join(paths.get_config_dir(), "play_history")
 CACHEFILE = os.path.join(paths.get_config_dir(), "cache_py_" + sys.version[0:5])
 READLINE_FILE = None
+categories = {
```

Comment:
The names were a big reason why I went with a simpler solution like this. 

Also, since that same list is used to classify normal videos, many of the categories may not be that applicable to live streams so we would clutter the output with a lot of potentially empty categories(since we're not polling to check that they have active channels)